cmd/dep/main.go

```diff
@@ -55,13 +55,44 @@ func main() {
 		fmt.Fprintln(os.Stderr)
 	}
 
-	if len(os.Args) <= 1 || len(os.Args) == 2 && (strings.Contains(strings.ToLower(os.Args[1]), "help") || strings.ToLower(os.Args[1]) == "-h") {
+	// parseArgs determines the name of the dep command and wether the user asked for
+	// help to be printed.
+	parseArgs := func(args []string) (cmdName string, printCmdUsage bool, exit bool) {
```

Comment:
I only left it as an anonymous function because I didn't have to leave main to implement this PR and parseArgs didn't seem  useful outside of `main`. I have no issue with moving it.